ios/brave-ios/Sources/Brave/Frontend/Browser/Toolbars/UrlBar/CollapsedURLBarView.swift

```diff
@@ -107,9 +107,13 @@ class CollapsedURLBarView: UIView {
         if let internalURL = InternalURL($0), internalURL.isBasicAuthURL {
           Strings.PageSecurityView.signIntoWebsiteURLBarTitle
         } else {
-          URLFormatter.formatURLOrigin(
-            forDisplayOmitSchemePathAndTrivialSubdomains: $0.absoluteString
-          )
+          if URLOrigin(url: $0).url == nil && URIFixup.getURL($0.absoluteString) == nil {
```

Comment:
What are trying to check here? `URLOrigin.url` being nil could mean the origin is opaque or it could mean the resulting `GURL` is not valid (the origin itself doesn't have a valid scheme/host/port tuple), or even the `GURL` to `NSURL` conversion failed. Why are we then checking `URIFixup`?